arch/x86/kvm/pkvm/pkvm.c

```diff
@@ -655,6 +657,13 @@ static bool is_guest_vcpu_accessible(struct kvm_vcpu *vcpu, enum pkvm_hc hc)
 		 * As the host needs to pre-configure the pVM's vCPU state for
 		 * booting, the protection for pVM is only enforced by the pKVM
 		 * hypervisor once the vCPU has started running.
+		 *
+		 * TODO: As the pVM can use another secure time source, the
```

Comment:
This TODO should be rather in the above comment for the `return true` case, not here?

Reply:
Makes sense. Will update.